validator/src/bin/solana-test-validator.rs

```diff
@@ -559,7 +568,11 @@ fn main() {
         genesis.compute_unit_limit(compute_unit_limit);
     }
 
-    match genesis.start_with_mint_address(mint_address, socket_addr_space) {
+    match genesis.start_with_mint_address(
+        mint_address,
+        socket_addr_space,
+        rpc_to_plugin_manager_receiver,
```

Comment:
I do not see this parameter being used here. Can you avoid changing this function for general testing as most cases they are not interested in this plugin stuff? If needed, create a new sub variant.

Reply:
good idea

Reply:
The ledger tool doesn't need it so I did a similar thing with `GeyserPluginService`